nixos/tests/nginx-sso.nix

```diff
@@ -27,18 +27,20 @@ import ./make-test.nix ({ pkgs, ... }: {
   };
 
   testScript = ''
-    startAll;
+    start_all()
 
-    $machine->waitForUnit("nginx-sso.service");
-    $machine->waitForOpenPort(8080);
+    machine.wait_for_unit("nginx-sso.service")
+    machine.wait_for_open_port(8080)
 
     # No valid user -> 401.
-    $machine->fail("curl -sSf http://localhost:8080/auth");
```

Comment:
the comments could be the titles of `subtest`s so this information is even visible in the log

Reply:
Done :)